quick-start-with-tidb.md

```diff
@@ -6,510 +6,532 @@ aliases: ['/docs/dev/quick-start-with-tidb/','/docs/dev/test-deployment-using-do
+For macOS:
 
-1. Download and install TiUP:
+```shell
+source ~/.zshrc
+```
 
-    {{< copyable "shell-regular" >}}
+</div>
+<div label="Linux">
 
-    ```shell
-    curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
-    ```
+For Linux:
 
-    If the following message is displayed, you have successfully installed TiUP:
-
-    ```log
-    Successfully set mirror to https://tiup-mirrors.pingcap.com
-    Detected shell: zsh
-    Shell profile:  /Users/user/.zshrc
-    /Users/user/.zshrc has been modified to add tiup to PATH
-    open a new terminal or source /Users/user/.zshrc to use it
-    Installed path: /Users/user/.tiup/bin/tiup
-    ===============================================
-    Have a try:     tiup playground
-    ===============================================
-    ```
+```shell
+source ~/.bash_profile
+```
 
-    Note the Shell profile path in the output above. You need to use the path in the next step.
+</div>
+</SimpleTab>
 
-2. Declare the global environment variable:
+## Start your database
 
-    > **Note:**
-    >
-    > After the installation, TiUP displays the absolute path of the corresponding Shell profile file. You need to modify `${your_shell_profile}` in the following `source` command according to the path. In this case, `${your_shell_profile}` is `/Users/user/.zshrc` from the output of Step 1.
+Launch a local TiDB cluster with one command:
 
-    {{< copyable "shell-regular" >}}
+```shell
+tiup playground
+```
 
-    ```shell
-    source ${your_shell_profile}
-    ```
+TiUP will automatically download the necessary binaries. Wait for the success message:
 
-3. Start the cluster in the current session:
+```log
+TiDB Playground Cluster is started, enjoy!
+```
 
-    - To start a TiDB cluster of the latest version with 1 TiDB instance, 1 TiKV instance, 1 PD instance, and 1 TiFlash instance, run the following command:
+Congratulations! You're running a distributed SQL database.
 
-        {{< copyable "shell-regular" >}}
+To proceed with the next steps, open another terminal and keep `tiup` running.
 
-        ```shell
-        tiup playground
-        ```
+## Connect to TiDB
 
-    - To specify the TiDB version and the number of instances of each component, run a command like this:
+You can connect to TiDB using either the built-in TiUP client or any MySQL-compatible client.
 
-        {{< copyable "shell-regular" >}}
+### Using TiUP client
 
-        ```shell
-        tiup playground v8.4.0 --db 2 --pd 3 --kv 3
-        ```
+The TiUP client gives you a convenient way to interact with TiDB via the command line.
 
-        The command downloads a version cluster to the local machine and starts it, such as v8.4.0. To view the latest version, run `tiup list tidb`.
+```shell
+tiup client
+```
 
-        This command returns the access methods of the cluster:
+Select the endpoint to connect, and try your first commands:
```

Comment:
```suggestion
Press Enter to connect to the endpoint, and try your first commands:
```

Reply:
Reason: It seems that pressing ↑ or ↓ is not working to switch the endpoint does not work. Users have to press Enter to connect to the default endpoint. 

![image](https://github.com/user-attachments/assets/f97ceb55-3e5e-4f0a-a554-80fe5e0652e2)